src/React/assets/dist/render_controller.js

```diff
@@ -1,25 +1,7 @@
 import React from 'react';
-import require$$0 from 'react-dom';
+import { createRoot } from 'react-dom/client';
```

Comment:
I updated `ux-react`'s importmap to require `react-dom/client` instead.

Reply:
Better add an entry in the React CHANGELOG then, to be safe ?

Reply:
You're right!

Reply:
Hum, in fact, thinking twice, this can be considered as a breaking change for people not using Symfony Flex (because the importmap won't be automatically updated).

I really prefer:
```js
import { createRoot } from 'react-dom/client';
```
over:
```js
import require$$0 from 'react-dom';

var createRoot;

var m = require$$0;
if (process.env.NODE_ENV === 'production') {
  createRoot = m.createRoot;
  m.hydrateRoot;
} else {
  var i = m.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED;
  createRoot = function(c, o) {
    i.usingClientEntryPoint = true;
    try {
      return m.createRoot(c, o);
    } finally {
      i.usingClientEntryPoint = false;
    }
  };
}
```

But the React package is not experimental, and I don't think we plan to release a v3.0 soon.

I will find a way to reuse `react-dom` in the built code

Reply:
Fixed by adding `'**/node_modules/react-dom/client.js'` to Rollup's plugin for TypeScript